net/unbound/files/unbound-odhcpd.defaults

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+[ "$(uci get dhcp.odhcpd.maindhcp)" = "1" ] || exit 0
+[ "$(uci get unbound.ub_main.dhcp_link)" = "none" ] || exit 0
+
+[ -x /usr/sbin/unbound-control ] && uci set unbound.ub_main.unbound_control='1'
+
+uci batch <<EOF
+set unbound.ub_main.add_local_fqdn='1'
+set unbound.ub_main.add_wan_fqdn='1'
+set unbound.ub_main.dhcp_link='odhcpd'
+set unbound.ub_main.dhcp4_slaac6='1'
+set unbound.ub_main.domain='lan'
+set unbound.ub_main.domain_type='static'
+set unbound.ub_main.listen_port='53'
```

Comment:
This is the default shipped in the package. If it is different, then you are messing with a configuration that is already customized. Again maybe a defensive check.

Reply:
Ok, the best would be then to move ALL default configuration into an `uci-defaults` script which exists right away if `/etc/config/unbound` already exists. In that way, we would be sure to never overwrite user configuration.